etc/kayobe/environments/ci-multinode/kolla/globals.yml

```diff
@@ -60,3 +61,10 @@ designate_ns_record:
 designate_backend: "bind9"
 designate_recursion: "yes"
 designate_forwarders_addresses: "1.1.1.1; 8.8.8.8"
+
+############################################################################
+# RabbitMQ
+
+# Ensure Rabbit is deployed with HA rather than quorum queues (to test migrations)
+om_enable_rabbitmq_high_availability: true
+om_enable_rabbitmq_quorum_queues: false
```

Comment:
We should use the default of Quorum in Caracal (i.e. just remove these lines). This was only set in Antelope so I could easily automate the mutinode upgrades up to Caracal

Reply:
Oops, missed that.